Elements/src/Serialization/JSON/JsonInheritanceConverter.cs

```diff
@@ -171,14 +171,15 @@ public override void WriteJson(Newtonsoft.Json.JsonWriter writer, object value,
 
         private static string GetDiscriminatorName(object value)
         {
-            var discriminatorName = "";
             var t = value.GetType();
-            discriminatorName += t.FullName.Split('`').First();
             if (t.IsGenericType)
             {
-                discriminatorName += "<" + String.Join(",", t.GenericTypeArguments.Select(arg => arg.FullName)) + ">";
+                return $"{t.FullName.Split('`').First()}<{String.Join(",", t.GenericTypeArguments.Select(arg => arg.FullName))}>";
```

Comment:
Very minor optimization. String interpolation is always recommended over string concatenation.